src/main/java/com/edu/ComNumber.java

```diff
@@ -0,0 +1,27 @@
+package com.edu;
+
+public class ComNumber {
+
+    public static final int COM_NUMBER_SIZE = 3;
+    public static final int MAX_COM_NUMBER_RANDOM = 9;
+    public static final int COM_NUMBER_START = 1;
```

Comment:
```suggestion
    private static final int COM_NUMBER_SIZE = 3;
    private static final int MAX_COM_NUMBER_RANDOM = 9;
    private static final int COM_NUMBER_START = 1;
```

클래스 내부에서만 쓰이는 상수이므로 접근 제한자를 ``private``으로 지정하는 것이 좋습니다. 
``자바 코드 컨벤션``에 따르면 또한 클래스의 이름은 **클래스 이름은 명사이어야 하며, 복합 단어일 경우 각 단어의 첫 글자는 대문자이어야 한다. 클래스 이름은 간단하고 명시적이 되도록 작성해라. 완전한 단어를 사용하고 두 문자어와 약어는 피하도록 하자 (만약, 약어가 URL이나 HTML과 같이 더 많이, 더 넓게 사용되고 있다면 약어를 사용하는 것도 괜찮다).** 라고 하고 있습니다. 

클래스명을 ``ComputerNumber``로 변경하는게 더 좋은 것 같습니다.